topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -87,17 +89,18 @@ We have used the HiCExplorer successfully with bwa, bowtie2 and hisat2. In this
 
 > ### {% icon hands_on %} Hands-on: Mapping reads
 >
-> 1. **Map with BWA-MEM 0.8.0** {% icon tool %}: Run Map with BWA-MEM on both strands `HiC_S2_1p_10min_lowU_R1` and `HiC_S2_1p_10min_lowU_R2` with:
+> 1. **Map with BWA-MEM 0.7.17.1** {% icon tool %}: Run Map with BWA-MEM on both strands `HiC_S2_1p_10min_lowU_R1` and `HiC_S2_1p_10min_lowU_R2` with:
 >    - "Will you select a reference genome from your history or use a built-in index?" to `Use a built-in index`
->    - "Select a reference genome" to `dm3`
->    - "Is this library mate-paired?" to `Single-end or interleaved paired-end`
+>    - "Using reference genome" to `dm6`

```

Comment:
It is dm3, please don't change this.

Reply:
hmm ok sure, when following the tutorial I was pointed to dm6 genes, and then thought it might be time to update.. https://github.com/galaxyproject/training-material/blame/master/topics/epigenetics/tutorials/hicexplorer/tutorial.md#L269 Will fix that link instead then.